server/data/prisonerSearchApiClient.ts

```diff
@@ -10,9 +11,16 @@ export default class PrisonerSearchApiClient {
   }
 
   async searchPrisoners(prisonerSearchCriteria: PrisonerSearchCriteria): Promise<Prisoner[]> {
-    return this.restClient.post({
+    return (await this.restClient.post({
```

Comment:
I dont think you need these awaits at this level - if you await at the calling function that should suffice